guides/common/modules/ref_operating-system-requirements.adoc

```diff
@@ -6,20 +6,25 @@
 The following operating systems are supported for deploying {Project}:
 
 .Operating systems supported for deploying {Project}
+[options="header"]
 |====
-| Operating system | Architecture | Notes
+| Operating system | Architecture
 ifdef::foreman-el,katello,orcharhino[]
-| {PlanningDocURL}Enterprise_Linux[{EL}] 9 | x86_64 only | EPEL is not supported.
+| {EL} 9 | x86_64 only
 endif::[]
 ifdef::satellite[]
-| {RHEL} 9 | x86_64 only |
+| {RHEL} 9 | x86_64 only
 endif::[]
 ifdef::foreman-deb[]
-| Debian 12 (Bookworm) | amd64 |
-| Ubuntu 22.04 (Jammy) | amd64 |
+| Debian 12 (Bookworm) | amd64
+| Ubuntu 22.04 (Jammy) | amd64
 endif::[]
 |====
 
+ifdef::foreman-el,katello,orcharhino[]
+Installing {Project} on a system with Extra Packages for {EL} (EPEL) is not supported.
```

Comment:
You should write out EL because on Satellite it becomes RHEL. However, the [EPEL](https://docs.fedoraproject.org/en-US/epel/) will always have a fixed name.
```suggestion
Installing {Project} on a system with Extra Packages for Enterprise Linux (EPEL) is not supported.
```